### DISCUSSION TITLE
support downloading videos from the Pandas Video platform

### DISCUSSION THREAD

**Discussion by itpropublic** [aclap-dev/video-downloadhelper#1436] [Help and Q&A]
### Your question

I have a quick question and hope someone can help. I'm using the Video DownloadHelper extension to capture some reference videos, but I'm unsure: does this extension support downloading videos from the Pandas Video platform? Has anyone tried this or has experience with this combination?

Thanks in advance for the help!

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You deleted the section you should have filled in when you created your problem report.  Go into your VDH settings & click the button that says `Copy information to clipboard`.  Then paste that into a post here.

Also, provide the URL of a sample video that you might like to download.  Don't say the Panda web site.  I've never heard of the Panda web site.  Make it easy for me to help you.  Post the URL of a specific video I can test with.
